packages/ibm-products/src/components/SteppedAnimatedMedia/SteppedAnimatedMedia.mdx

```diff
@@ -0,0 +1,22 @@
+import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
```

Comment:
Given our current Storybook 7 configuration, move these docs to `SteppedAnimatedMedia.docs-page.js` and delete this file.